extensions/frontend/pkg/revaconfig/config.go

```diff
@@ -167,6 +167,7 @@ func FrontendConfigFromStruct(cfg *config.Config) map[string]interface{} {
 								"default_permissions":               22,
 								"search_min_length":                 3,
 								"public": map[string]interface{}{
+									"alias":                      true,
```

Comment:
Naming can be discussed. Suggestions welcome!

Reply:
When redefining capabilities with @C0rby and @wkloucek we often ended up using a simple object, e.g. `{version: x, enabled: false, available: true}` for more flexibility. Not sure how applicable that is in this case

Also, not sure what the ideal naming should be (as hinted in the ticket) - "alias" doesn't really give away the fact that this is an internal-link?

Reply:
As in "I think alias-link was the oc-internal development title for the functionality, not sure what e.g. an admin user expects in regards of naming"

Reply:
@kobergj formatting seems off and has a conflicting file, could you follow up here? 😇

Reply:
So sorry. Completely forgot :see_no_evil: